### PR TITLE
Added mailchimp connector (add member)

### PR DESCRIPTION
**Pull request by AbdulTheActivePiecer**

### CODE REVIEW

File: packages/pieces/src/apps/mailchimp/actions/add-member-to-list.action/add-member-to-list.action.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+export const addMemberToList = createAction({
+    name: 'add_member_to_list',
+    displayName: "Add Member to List",
+    description: "Add a member to an existing Mailchimp list",
+    props: {
+        authentication: Property.OAuth2({
+            description: "",
+            displayName: 'Authentication',
+            authUrl: "https://login.mailchimp.com/oauth2/authorize",
+            tokenUrl: "https://login.mailchimp.com/oauth2/token",
```
Comment (abuaboud) [inline packages/pieces/src/apps/mailchimp/actions/add-member-to-list.action/add-member-to-list.action.ts]: Please move the common properties to common folder, same for base Url.

That would make it easier to write more actions / trigger

File: packages/pieces/src/apps/mailchimp/actions/add-member-to-list.action/add-member-to-list.action.ts [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+async function getUserLists(authProp: AuthPropertyValue): Promise<{lists:MailChimpList[]}> {
+    const access_token= authProp.access_token;
+    console.log(authProp);
+        const mailChimpMetaDataRequest: HttpRequest<{dc:string}> = {
+            method: HttpMethod.GET,
+            url: 'https://login.mailchimp.com/oauth2/metadata',
+            headers: {
+                Authorization: `OAuth ${access_token}`
+              }
+        };
```
Comment (abuaboud) [inline packages/pieces/src/apps/mailchimp/actions/add-member-to-list.action/add-member-to-list.action.ts]: This can be moved to an common helper function.

**Review by abuaboud — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -18,9 +18,6 @@ dist/
 coverage/
 node_modules/
 
-# worker in backend resources
```
Comment (abuaboud) [inline .gitignore:21]: why this is removed?

**Review by abuaboud — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 3 above)
Comment (AbdulTheActivePiecer) [reply .gitignore:21]: src/backend/backend-loader/src/main/resources/activepieces-worker.js this is from old backend

File: packages/pieces/src/apps/mailchimp/actions/add-member-to-list.action/add-member-to-list.action.ts (hunk 1 above)
Comment (AbdulTheActivePiecer) [reply packages/pieces/src/apps/mailchimp/actions/add-member-to-list.action/add-member-to-list.action.ts]: done

File: packages/pieces/src/apps/mailchimp/actions/add-member-to-list.action/add-member-to-list.action.ts (hunk 2 above)
Comment (AbdulTheActivePiecer) [reply packages/pieces/src/apps/mailchimp/actions/add-member-to-list.action/add-member-to-list.action.ts]: done

**Review by abuaboud — APPROVED** [review]

**Pull request merged**